webtau-core/src/main/java/com/twosigma/webtau/reporter/TestListener.java

```diff
@@ -5,11 +5,40 @@
  * Core usage is to print current test name
  */
 public interface TestListener {
-    void beforeFirstTest();
+    /**
+     * before any test is ran
+     */
+    default void beforeFirstTest() {}
 
-    void beforeTestRun(WebTauTest test);
+    /**
+     * before test code is invoked
+     * @param test test
```

Comment:
these param lines don't add much to the code -- can we remove them or is some nit-picky style checker going to complain?

Reply:
I believe javadoc will fail

Reply:
Javadoc will fail if you only do part of the javadoc.  If you don't have the comment at all, everything will be awesome.